src/index.html

```diff
@@ -182,37 +199,13 @@ <h1 class="settings-sub-heading">Features</h1>
         </div>
         <p>Sorry, wasn't ready…</p>
       </div>
+      <link rel="preload" as="script" href="js/page.js" onload="appendScript(href)">
       <script>
         document.querySelector('.preloader').addEventListener('click', function() {
           // track it
           ga('send', 'event', 'preloader', 'clicked');
           this.className = 'preloader active';
         });
-
-        requestAnimationFrame(function() {
-          requestAnimationFrame(function() {
-            [
-              'all.css',
-              'https://fonts.googleapis.com/css?family=Roboto:400,700%7CInconsolata'
-            ].forEach(function(url) {
-              var link = document.createElement('link');
-              link.rel = 'stylesheet';
-              link.crossorigin = true;
```

Comment:
Hah, this was doing nothing, since it's `crossOrigin` not `crossorigin`.

Reply:
See, it's good fiddling with things that haven't been touched for a long time, after all! 😛